From khali@linux-fr.org Mon Jun 12 12:46:10 2006
Date: Mon, 12 Jun 2006 21:46:04 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: LM Sensors <lm-sensors@lm-sensors.org>, Jordan Crouse <jordan.crouse@amd.com>
Subject: [PATCH 04/14] scx200_acb: Mark scx200_acb_probe __init
Message-Id: <20060612214604.3050e740.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-scx200_acb-mark-scx200_acb_probe-init.patch

In i2c bus driver scx200_acb, function scx200_acb_probe can be
tagged __init.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/scx200_acb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/busses/scx200_acb.c
+++ gregkh-2.6/drivers/i2c/busses/scx200_acb.c
@@ -384,7 +384,7 @@ static struct i2c_algorithm scx200_acb_a
 static struct scx200_acb_iface *scx200_acb_list;
 static DECLARE_MUTEX(scx200_acb_list_mutex);
 
-static int scx200_acb_probe(struct scx200_acb_iface *iface)
+static __init int scx200_acb_probe(struct scx200_acb_iface *iface)
 {
 	u8 val;
 
